src/Adaptor.js

```diff
@@ -227,15 +228,17 @@ export function insertMany(table, records) {
  * upsert(
  *  'users', // the DB table
  *  'ON CONSTRAINT users_pkey', // a DB column with a unique constraint OR a CONSTRAINT NAME
- *  {name: 'Elodie', id: 7}
+ *  {name: 'Elodie', id: 7},
+ *  {writeSql:true, execute: true}
```

Comment:
line 232 should be `{writeSql, execute: true}` because (a) a space is missing and (b) providing the key without a value should create `writeSql: true`.